packages/pyinstaller/bitmessagemain.spec

```diff
@@ -72,6 +73,13 @@ a.datas += [
     for file_ in os.listdir(dir_append) if file_.endswith('.ui')
 ]
 
+sql_dir_append = glob(os.path.join(srcPath, 'sql/*.sql'))
```

Comment:
this whole section can be replaced with glob, like in the other files

Reply:
Is this like?

```a.datas += os.path.join(srcPath, 'sql'), glob(os.path.join(srcPath, 'sql/*.sql')
```